ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-missing-evpn-multicast-l3-with-pim.yml

```diff
@@ -43,4 +43,4 @@ tenants:
 
 expected_error_message: >-
   'pim: enabled' set on l3_interface 'Ethernet1' on 'failure-missing-evpn-multicast-l3-with-pim'
-  requires evpn_l3_multicast: enabled: true under VRF 'Tenant_A_OP_Zone_MC' or Tenant 'FABRIC'
+  requires 'evpn_l3_multicast: enabled: true' under VRF 'Tenant_A_OP_Zone_MC' or Tenant 'FABRIC'
```

Comment:
Something looks off here. should be `evpn_l3_multicast.enabled: true`

Reply:
that was the existing error message - happy to change it